commit 96aaf5ca9a512382c9f1ba4bd9053c82153b75c0 (refs/changes/80/5380/4)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-01-18T08:38:00+00:00 (9 months ago)
    
    TRITON-633 dangling symlinks break the adminui build when using /usr/bin/cp
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/www/img/glyphicons-halflings-white.png b/www/img/glyphicons-halflings-white.png
deleted file mode 120000
index 08fa0f2b..00000000
--- a/www/img/glyphicons-halflings-white.png
+++ /dev/null
@@ -1 +0,0 @@
-../../bootstrap/img/glyphicons-halflings-white.png
\ No newline at end of file
diff --git a/www/img/glyphicons-halflings.png b/www/img/glyphicons-halflings.png
deleted file mode 120000
index 10ff0466..00000000
--- a/www/img/glyphicons-halflings.png
+++ /dev/null
@@ -1 +0,0 @@
-../../bootstrap/img/glyphicons-halflings.png
\ No newline at end of file
